content/using_ec2_spot_instances_with_eks/prerequisites/k8stools.md

```diff
@@ -16,7 +16,7 @@ for the download links.](https://docs.aws.amazon.com/eks/latest/userguide/gettin
 #### Install kubectl
 
 ```
-export KUBECTL_VERSION=v1.18.12
+export KUBECTL_VERSION=v1.19.6
```

Comment:
You need to upgrade cluster autoscaler also to 1.19.X. It's 1.18.3.

- image: us.gcr.io/k8s-artifacts-prod/autoscaling/cluster-autoscaler:v1.18.3

Also try to fix the roles related depreciation warnings for cluster autoscaler. 

Admin:~/environment $ kubectl apply -f ~/environment/cluster-autoscaler/cluster_autoscaler.yml
serviceaccount/cluster-autoscaler created
Warning: rbac.authorization.k8s.io/v1beta1 ClusterRole is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 ClusterRole
clusterrole.rbac.authorization.k8s.io/cluster-autoscaler created
Warning: rbac.authorization.k8s.io/v1beta1 Role is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 Role
role.rbac.authorization.k8s.io/cluster-autoscaler created
Warning: rbac.authorization.k8s.io/v1beta1 ClusterRoleBinding is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 ClusterRoleBinding
clusterrolebinding.rbac.authorization.k8s.io/cluster-autoscaler created
Warning: rbac.authorization.k8s.io/v1beta1 RoleBinding is deprecated in v1.17+, unavailable in v1.22+; use rbac.authorization.k8s.io/v1 RoleBinding
rolebinding.rbac.authorization.k8s.io/cluster-autoscaler created
deployment.apps/cluster-autoscaler created

Reply:
Moved the cluster-autoscaler to v1.20.0 and fixed the depreciation warnings.

Reply:
No, don't just use the latest cluster-autoscaler version. you need to match between the K8s and the CA major versions because it's not recommended to mix major versions. so you need the latest CA from major 1.19 which at this time is 1.19.1. Also according to the release the proper URL for the image is 
k8s.gcr.io/autoscaling/cluster-autoscaler:v1.19.1
and not
us.gcr.io
I believe the former is a geo routing based target to increase performance and resilience of the image download

Reply:
Changed to use the Cluster Autoscaler version 1.19.1 which is compatible with Kubernetes version 1.19. Updated the release URL to k8s.gcr.io/autoscaling/cluster-autoscaler:v1.19.1.